### DISCUSSION TITLE
2024.03.13(수)

### DISCUSSION THREAD

**Discussion by Crack-Egg** [Party-Time-Job/Party-Time-Job#41] [회의록]
## 1. 머지 규칙

PR 올린 당사자만 머지하기

## 2. 페이지 이동

페이지 이동은 feature 폴더에서 구현하기

## 3. 타입 지정

타입은 API를 받아오는 경우는 타입 파일을 따로 만들어서 관리하고, 컴포넌트 타입은 함수 표현 위에 작성하는 것으로 통일

## 4. 깃허브 디스코드 알림 요청

PR 요청 및 코드 리뷰만 설정

## 5. 발표자

치현님!!
데드라인 2일 전에 발표 자료 준비, 대본 작성, 발표자 등 역할 분배하기

**Comment by DanteSnow** [comment]
추가 내용
1. 폼 -> 리액트 훅 폼 통일하기
2. 타입, API -> FSD 폴더구조 따르기로 결정

- 엔티티, 피쳐 나누는 기준 -> 데이터를 화면에 렌더링 하는지
